eval2.py

```
@@ -0,0 +1,258 @@
+    plt.grid(True, alpha=0.3)
+    
+    plt.tight_layout()
+    output_path = os.path.join(output_dir, filename)
+    plt.savefig(output_path, dpi=300, bbox_inches='tight')
+    plt.close()
+    print(f"Saved boxplot to: {output_path}")
+
+def load_model(model_path, model_info_path, device):
+    """Load model from checkpoint"""
+    with open(model_info_path, 'r') as f:
+        model_info = json.load(f)
```

Ruff (0.8.2)

Local variable `model_info` is assigned to but never used

Remove assignment to unused variable `model_info`

(F841)